src/pages/MetricDetail.svelte

```diff
@@ -204,6 +206,14 @@
       <td>Version</td>
       <td>{metric.version || 0}</td>
     </tr>
+    {#if getSearchfoxLink(params.app, metric.name)}
+      <tr>
+        <td>Searchfox</td>
+        <td>
+          <a href={getSearchfoxLink(params.app, metric.name)}>{metric.name}</a>
+        </td>
```

Comment:
I feel like maybe we should add some kind of warning if the metric is expired here, but we can do that in a followup.

Reply:
:+1:

Reply:
I'll add the warning in a followup PR after #341 has landed

Reply:
Should I still add a warning here @wlach, or do you think the expired Pill is sufficient enough?

Reply:
I think it would be worth adding a quick warning. Doesn't hurt.